plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsStorage.java

```diff
@@ -58,15 +57,11 @@ public class TestDeltaLakeAdlsStorage
 
     private HiveHadoop hiveHadoop;
 
-    @Parameters({
-            "hive.hadoop2.azure-abfs-container",
-            "hive.hadoop2.azure-abfs-account",
-            "hive.hadoop2.azure-abfs-access-key"})
-    public TestDeltaLakeAdlsStorage(String container, String account, String accessKey)
+    public TestDeltaLakeAdlsStorage()
     {
-        requireNonNull(container, "container is null");
-        this.account = requireNonNull(account, "account is null");
-        this.accessKey = requireNonNull(accessKey, "accessKey is null");
+        String container = requireNonNull(System.getProperty("hive.hadoop2.azure-abfs-container"), "container is null");
```

Comment:
if the property is not set, the exception message should contain the name of the property

Reply:
I will add validation after tests are executed (I don't want to lose the spot)